incidents/2020-05-20-seach-outage.md

```diff
@@ -0,0 +1,66 @@
+# 2020-05-20 Search Outage
```

Comment:
to be more precise, search wasn't out of service but it stopped accepting writes